assembl/static2/js/app/utils/translations.js

```diff
@@ -285,11 +285,24 @@ const Translations = {
       userName: "Nom d'utilisateur",
       fullname: "Nom complet",
       email: "Email",
+      oldPassword: "Ancien mot de passe",
+      newPassword: "Nouveau mot de passe",
+      newPassword2: "Retaper le mot de passe",
       memberSince: "Membre depuis le %{date}",
       save: "Enregistrer",
       password: "Mot de passe",
-      changePassword: "Modifier",
-      saveSuccess: "Votre profil a été mis à jour avec succès"
+      changePassword: "Modifier mon mot de passe",
+      passwordModifiedSuccess: "Votre mot de passe a été modifié avec succès",
+      saveSuccess: "Votre profil a été mis à jour avec succès",
+      updateUser: {
+        errorMessage: {
+          1: "Nous avons déjà un utilisateur ayant ce nom d'utilisateur.",
```

Comment:
Yeah, so these could be done via the backend already, reducing the json file size and computation on front-end.

Reply:
Well, for 4 more messages, it doesn't make a lot of difference. If you really want to do something about the bundle size, we probably should load a json file in async when you change the language interface.

Reply:
Yah, it's 4 messages now. But now this is the start of a pattern for handling errors. Yourself and other developers would pick this up and make it the norm of exception exchange between boundaries.

Reply:
I'm not against revisiting this. But we need to fix the selected language mismatch between frontend and backend first.

Reply:
That will likely be resolved with https://bluenove.atlassian.net/browse/DEVAS-715.

Reply:
Yes, this is this issue.